packages/x-charts/src/internals/plugins/corePlugins/useChartSeries/useChartSeries.ts

```diff
@@ -49,8 +49,8 @@ useChartSeries.params = {
 const EMPTY_ARRAY: any[] = [];
```

Comment:
Maybe we need to change this to `ReadonlyArray<any>`?

Reply:
Not needed as this `EMPTY_ARRAY` is used on the internal type, which is mutable